refinery/core/views.py

```diff
@@ -177,7 +178,10 @@ def user(request, query):
     except User.DoesNotExist:
         user = get_object_or_404(UserProfile, uuid=query).user
 
-    return render_to_response('core/user.html', {'profile_user': user},
+    # return all non-manager groups in profile
+    groups = user.groups.exclude(name__contains='Managers')
```

Comment:
How about `[group for group in user.groups if not group.is_manager_group()]`

I doubt it would change anytime soon, but relying on the name containing `Managers` could eventually fail without us knowing

Reply:
I'll add a unit test

Reply:
@scottx611x I moved it to a utils method due to issues accessing context in unit tests.